src/socketio.quiver.js

```diff
@@ -529,13 +560,21 @@ QuiverSocialProvider.prototype.inviteUser = function(ignoredUserId, cb) {
     if (!server) {
       cb(undefined, this.err('Can\'t invite without a valid connection'));
     }
-    /** @type QuiverSocialProvider.invite_ */
-    var invite = {
-      servers: [server],
-      userId: this.configuration_.self.id,
-      nick: this.configuration_.self.nick
-    };
-    cb({networkData: JSON.stringify(invite)});
+
+    // 16 bytes is the standard for collision avoidance in a UUID.
```

Comment:
It would be good to comment that we need knockCodes for Quiver to ensure that we don't leak info (e.g. server list) to unknown clients.